go.mod

```diff
@@ -13,62 +13,65 @@ require (
 )
 
 require (
-	github.com/Microsoft/go-winio v0.6.0 // indirect
```

Comment:
What triggered these updates?

Reply:
When trying to compile, I ran `go mod tidy` after some build issues

Reply:
I just ran `go mod tidy` and `make build` on main and nothing changed. Please rerun `go mod tidy` and see if anything changes. Something strange if it doesn't.

Reply:
@ChristianDarr-personal Yes these changes seem to be not necessary and not sure how it happened. So you can just delete or comment out the indirect ones and run `go mod tidy` and that should restore the old dependencies back.